CONFIGURATION.md

```diff
@@ -182,9 +182,11 @@ regexp: <regex>,
 [ source_ip_address: <string> ]
 
 # The query sent in the TCP probe and the expected associated response.
+# expect_bytes does exact byte-by-byte match.
```

Comment:
can we mention that `expect` and `expect_bytes` are mutually exclusive?

we do mention it for `body` and `body_file` in http_probe, and few other places.